client/src/component/Sign-Up-Page.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
+
```

Comment:
You will need axios to do your post requests to the database through the endpoints we setup.